packages/dev/core/src/Materials/Node/Blocks/PBR/pbrMetallicRoughnessBlock.ts

```diff
@@ -282,6 +282,12 @@ export class PBRMetallicRoughnessBlock extends NodeMaterialBlock {
     @editableInPropertyPage("Radiance occlusion", PropertyTypeForEdition.Boolean, "ADVANCED", { notifiers: { update: true } })
     public useRadianceOcclusion: boolean = true;
 
+    /**
+     * Use a different model for rough radiance that tends to better match raytracing.
+     */
+    @editableInPropertyPage("Alternate Rough Radiance", PropertyTypeForEdition.Boolean, "ADVANCED", { embedded: true, notifiers: { update: true } })
+    public useAlternateRoughRadiance: boolean = false;

```

Comment:
I don't really like the "Alternate Rough Radiance" naming as it isn't very descriptive. Any better ideas?

Reply:
Asking the team, I will let you know if the name should be updated.

Reply:
Also are we sure there will only ever be two modes? Would it make more sense to make this a mode with more descriptive names? Something like
```ts
export type RoughRadianceMode = "default" | "mixWithIrradiance";
...
public roughRadianceMode: RoughRadianceMode = "default";
```

Reply:
It's very hard to say...

If we go that way, we'd practically have to update all our Boolean properties in case there are more options in the future, which isn't really feasible.

Let's hope we don't have any new options in the future! :)

Reply:
maybe `mixRoughRadianceWithIrradiance` as at least it is hghly descriptive ?